### ISSUE TITLE
Selection deletion bug

### ISSUE THREAD

**Issue by danocmx** [ZeusJunior/tf2-automatic-gui#38]
```js
Promise {
  <rejected> TypeError: Cannot read property 'sku' of undefined
      at fs.readJSON.then (C:\Users\Glen\Documents\Code\tf2-automatic-gui\app\pricelist.js:232:23) }
```

**Comment by ZeusJunior** [comment]
Maybe like #34 ?

**Comment by ZeusJunior** [comment]
Fixed in dev, will be released in v1.7.1
